.github/workflows/bundle-size-base.yml

```diff
@@ -10,29 +10,21 @@ concurrency:
   cancel-in-progress: true
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33202&commitId=bb215164c324cf80ff64ff9edcb0849f92f5e731&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|9| Changed|

</details>


  <details><summary><b>Drawer</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|2244| Changed|
|Drawer.overlay drawer full.chromium.png|1137| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|991| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>